<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgGeneralSettings</class>
 <widget class="QWidget" name="WdgGeneralSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>340</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="textLabel1">
     <property name="text">
      <string>Cursor shape:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QComboBox" name="m_cmbCursorShape"/>
   </item>
   <item row="0" column="3">
    <spacer name="spacer1">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>41</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QCheckBox" name="m_showOutlinePainting">
     <property name="text">
      <string>Show outline while painting</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="chkShowRootLayer">
     <property name="text">
      <string>Show root layer</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="m_autosaveCheckBox">
     <property name="text">
      <string>Autosave every:</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QSpinBox" name="m_undoStackSize">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="singleStep">
      <number>5</number>
     </property>
     <property name="value">
      <number>30</number>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QSpinBox" name="m_autosaveSpinBox">
     <property name="suffix">
      <string notr="true">min</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1440</number>
     </property>
     <property name="singleStep">
      <number>5</number>
     </property>
     <property name="value">
      <number>15</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Undo stack size:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QCheckBox" name="m_backupFileCheckBox">
     <property name="text">
      <string>Create backup file </string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <spacer name="spacer2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>41</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Favorite presets:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QSpinBox" name="m_favoritePresetsSpinBox">
     <property name="minimum">
      <number>10</number>
     </property>
     <property name="maximum">
      <number>30</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_autosaveCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_autosaveSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
